cluster-controller/src/main/java/io/strimzi/controller/cluster/model/KafkaCluster.java

```diff
@@ -121,12 +156,13 @@ public static KafkaCluster fromConfigMap(ConfigMap kafkaClusterCm) {
                 Labels.fromResource(kafkaClusterCm));
 
         Map<String, String> data = kafkaClusterCm.getData();
+        checkAll(data);
         kafka.setReplicas(Integer.parseInt(data.getOrDefault(KEY_REPLICAS, String.valueOf(DEFAULT_REPLICAS))));
```

Comment:
So here at the call site we'd have

```
kafka.setReplicas(getInteger(data, KEY_REPLICAS, DEFAULT_REPLICAS));
```

Which is shorter, more readable, and avoids the slightly crazy convert-`DEFAULT_REPLICAS`-to-a-string-so-we-can-call-`parseInteger`-with-it.